contracts/governance/extensions/GovernorTimelockControl.sol

```diff
@@ -17,7 +17,7 @@ import {SafeCast} from "../../utils/math/SafeCast.sol";
  * the assets and permissions must be attached to the {TimelockController}. Any asset sent to the {Governor} will be
  * inaccessible from a proposal, unless executed via {Governor-relay}.
  *
- * WARNING: Setting up the TimelockController to have additional proposers or cancellers besides the governor is very
+ * WARNING: Setting up the TimelockController to have additional proposers or cancelers besides the governor is very
```

Comment:
`canceller` and `canceler` are both correct spellings but 2 l's is more common in British English while the other is American English. We should just be consistent on what we choose.